Jorge/HTML & CSS/styles.css

```diff
@@ -0,0 +1,142 @@
+body {
+  background-color: lightblue;
+  display: flex;
+  justify-content: center;
+}
+.main-container {
+  background-color: grey;
+  width: 80%;
+}
+.header {
+  display: flex;
+  background-color: white;
+  margin: 1rem;
```

Comment:
lets try to avoid using relative values in margins to reduce the chance of having inconsistent styling across browsers, or margins, paddings lets stick with px, preferably values that are divisible by 8. If you need a smaller value you can use 4px.